src/System.ServiceModel.UnixDomainSocket/src/System/ServiceModel/UnixDomainSocketBinding.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.ServiceModel.Channels;
+using System.Xml;
+
+namespace System.ServiceModel
+{
+    public class UnixDomainSocketBinding : Binding
+    {
+        //private OptionalReliableSession _reliableSession;
+        // private BindingElements
+        private UnixDomainSocketTransportBindingElement _transport;
+        private BinaryMessageEncodingBindingElement _encoding;
+       // private ReliableSessionBindingElement _session;
+        private UnixDomainSocketSecurity _security = new UnixDomainSocketSecurity();
+
+        public UnixDomainSocketBinding()
+        {
+            Initialize();
+        }
+
+        public UnixDomainSocketBinding(SecurityMode securityMode)
+            : this()
+        {
+            _security.Mode = securityMode;
+        }
+
+        public UnixDomainSocketBinding(string configurationName)
+            : this()
```

Comment:
Remove this method. It's only in the other bindings for legacy reasons. This is a new type so no need to start with this unsupported constructor.